app/code/Magento/Search/view/frontend/web/form-mini.js

```diff
@@ -88,7 +88,7 @@ define([
                 }
 
                 setTimeout($.proxy(function () {
-                    if (this.autoComplete.is(':hidden')) {
+                    if (this.autoComplete.css('display', 'none')) {
```

Comment:
Looks like this line will change display to "none", that is not expected there

Reply:
Looks like you are right, I'll still working it!